stubs/bleach/bleach/html5lib_shim.pyi

```diff
@@ -1,27 +1,59 @@
-from _typeshed import Incomplete
-from collections.abc import Generator, Iterable
+import re
+from collections.abc import Iterable, Iterator
+from typing import Any
 
-class HTMLParser:  # actually html5lib.HTMLParser
-    def __getattr__(self, __name: str) -> Incomplete: ...
+# We don't re-export any `html5lib` types / values here, because they are not
+# really public and may change at any time. This is just a helper module,
+# import things directly from `html5lib` instead!
+from html5lib import HTMLParser
+from html5lib._inputstream import HTMLUnicodeInputStream
+from html5lib._tokenizer import HTMLTokenizer
+from html5lib.serializer import HTMLSerializer
 
-class Filter:  # actually html5lib.filters.base.Filter
-    def __getattr__(self, __name: str) -> Incomplete: ...
+HTML_TAGS: list[str]
+HTML_TAGS_BLOCK_LEVEL: frozenset[str]
+AMP_SPLIT_RE: re.Pattern[str]
+ENTITIES: dict[str, str]
+TAG_TOKEN_TYPES: set[int]
+TAG_TOKEN_TYPE_CHARACTERS: int
+TAG_TOKEN_TYPE_END: int
+TAG_TOKEN_TYPE_PARSEERROR: int
+TAG_TOKEN_TYPE_START: int
 
-class SanitizerFilter:  # actually html5lib.filters.sanitizer.Filter
-    def __getattr__(self, __name: str) -> Incomplete: ...
+class InputStreamWithMemory:
+    position: int
```

Comment:
`position` is assigned here, and it's whatever `self._inner_stream.position` is: https://github.com/mozilla/bleach/blob/6cd4d527a6b43569c1e1490e632500199b1efb6c/bleach/html5lib_shim.py#L250

`self._inner_stream` is an `html5lib._inputstream.HTMLUnicodeInputStream` object according to your `__init__` annotation, and `HTMLUnicodeInputStream.position` is a method: https://github.com/html5lib/html5lib-python/blob/f7cab6f019ce94a1ec0192b6ff29aaebaf10b50d/html5lib/_inputstream.py#L229-L232